lib/matplotlib/axes/_axes.py

```diff
@@ -4361,9 +4365,10 @@ def merge_kw_rc(subkey, explicit, zdelta=0, usemarker=True):
             vert = mpl.rcParams['boxplot.vertical']
         else:
             _api.warn_deprecated(
-                "3.10",
+                "3.11",
```

Comment:
I would have expected 3.10 here („pending depreciation in 3.10“) but am not sure what the message looks like („deprecated in 3.11“) would also be fine.

Reply:
The version number actually does not appear in the Pending deprecation, apparently:

```
PendingDeprecationWarning: vert: bool will be deprecated in a future version. Use orientation: {'vertical', 'horizontal'} instead.
```

Personally I think this is fine, and leaving the intended deprecation version in source reduces change when actually deprecating (though in this case, it is already fully deprecated on main, so that much doesn't really matter)